src/infrastructure/Http.ts

```diff
@@ -79,6 +79,9 @@ export abstract class Http {
         if (error.code && error.address && error.code === 'ECONNREFUSED') {
             return new Error(`Cannot reach node: ${error.address}:${error.port}`);
         }
+        if (error instanceof Error) {
+            return error;
+        }
         return new Error(error);
     }
```

Comment:
If we just wrap the exception into a new one Error, the original stack trace is lost. We want the stack trace when something in our code fails, like an NPE when mapping the DTO.